### ISSUE TITLE
Checkbox styles slightly inconsistent in NcActionCheckbox vs NcCheckboxRadioSwitch

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud-libraries/nextcloud-vue#4893]
| NcActionCheckbox | NcCheckboxRadioSwitch |
|--------|--------|
| ![Bildschirmfoto vom 2023-11-30 08-05-20](https://github.com/nextcloud-libraries/nextcloud-vue/assets/1374172/7ece48b0-487d-4a7a-915b-b61d168d603f) | ![Bildschirmfoto vom 2023-11-30 08-05-29](https://github.com/nextcloud-libraries/nextcloud-vue/assets/1374172/282fc8ed-dc91-47ce-a9bb-350de4b3f52c) | 

Left is smaller and thinner with smaller border radios.

**Comment by ShGKme** [comment]
Would you prefer to have style updated in NcAction?

**Comment by jancborchardt** [comment]
Yup, NcAction should be updated, the slightly thicker ones are the new design. :)
